assets/js/components/ReportTable.js

```diff
@@ -89,64 +103,79 @@ export default function ReportTable( props ) {
 				</thead>
 
 				<tbody className="googlesitekit-table__body">
-					{ ! rows?.length && ZeroState && (
+					{ gatheringData && zeroDataStatesEnabled && (
 						<tr className="googlesitekit-table__body-row">
 							<td
 								className="googlesitekit-table__body-item"
 								colSpan={ columns.length }
 							>
-								<ZeroState />
+								<GatheringDataNotice />
 							</td>
 						</tr>
 					) }
+					{ ! gatheringData &&
+						zeroDataStatesEnabled &&
+						! rows?.length &&
+						ZeroState && (
```

Comment:
Shouldn't it be "if ( **not zeroDataStatesEnabled** or **not gatheringData** ) and empty rows and not empty ZeroState"?

Reply:
Nice catch! The confusion of `gatheringData` depending on `zeroDataStatesEnabled` caught us out here. Hence I was just setting `gatheringData` to false if `zeroDataStatesEnabled` was false to avoid confusion. 😄